v22.1/backup-architecture.md

```diff
@@ -0,0 +1,101 @@
+
+- Test the connection to the storage bucket URL (`'s3://bucket'`).
+- Map out the directory to which the nodes will write.
+- Gather all the leaseholders of the keys to be backed up.
+
+To map out the directory to which the nodes will write, the coordinator identifies the [type](backup-and-restore-overview.html#backup-and-restore-types) of backup. This determines the name of the new (or edited) directory to store the backup files in. For example, if there is a full backup existing in the target storage location, the upcoming backup will be [incremental](take-full-and-incremental-backups.html#incremental-backups) and therefore append to the [full](take-full-and-incremental-backups.html#full-backups) backup.
+
+From the cluster, the coordinator will gather all the [leaseholders](architecture/reads-and-writes-overview.html) of the keys that will be part of the backup. The leaseholder is responsible for serving the data from the storage layer because all [writes](architecture/reads-and-writes-overview.html#write-scenario) go through a leaseholder. Since any node in a cluster can become the coordinator and all nodes could be responsible for exporting data during a backup, it is necessary that all nodes can connect to the storage location.
+
+In the following diagram, **Node 2** and **Node 3** contain the leaseholders for the **R1** and **R2** ranges. Therefore, for the example backup job, these nodes will export the backup data to the specified storage location.
+
+<img src="{{ 'images/v22.1/backup-processing.png' | relative_url }}" alt="Three-node cluster exporting backup data from the leaseholders" style="border:0px solid #eee;max-width:100%" />
+
+While processing, the nodes write checkpoint progress to track their backup work. The coordinating node will aggregate the progress data into checkpoint files in the storage bucket. The checkpoint files provide a marker for the backup to resume after a retryable state, such as when it has been paused.
+
+The coordinator will begin to write the backup manifest file only after each of the nodes have completed their work. In the diagram, **Node 2** is exporting the backup data for **R1** as its leaseholder as well as exporting the backup's metadata as the coordinator.
+
+The backup manifest file is a metadata representation of everything a backup contains. That is, all the information a [restore](restore.html) job will need to complete successfully. A backup without a manifest file would indicate that the backup did not complete properly and would not be restorable.
+
+With multiple backups complete, the specified storage location will contain a set of backups, which CockroachDB defines as a [_backup collection_](take-full-and-incremental-backups.html#backup-collections).
```

Comment:
This line could be a little confusing: are we talking about _a_ backup to _a_ location, or are we talking about backing up to a _collection_ repeatedly, i.e. performing subsequent backups? architecturally, those subsequent backups are to different locations i.e. unique paths within the collection. 

I think I'd maybe just make a whole separate section for collections?